src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -86,6 +86,11 @@ public function load(array $configs, ContainerBuilder $container)
         if ($config['enable_fos_user']) {
             $loader->load('fos_user.xml');
         }
+
+        // NelmioApiDoc support
+        if ($config['nelmio_api_doc']) {
```

Comment:
Yes, what would be nice is to have both. The enable flag can be `true` by default, but the load of `nelmio_api_doc.xml` skipped if the bundle isn't registered.

It will enable the doc support if both bundles are registered but give the ability to the user to disable doc support if he wants.

Btw, wdyt about renaming the config option `enable_nelmio_api_doc` to be coherent with the FOSUser flag?